<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileView</class>
 <widget class="QDialog" name="ProfileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人资料</string>
  </property>
  <widget class="QGroupBox" name="gbProfile">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>450</width>
     <height>325</height>
    </rect>
   </property>
   <property name="title">
    <string>修改账号信息</string>
   </property>
   <widget class="QLabel" name="lblID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>25</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhone">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>用户电话号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>175</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>用户密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDeux">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>225</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePassword">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>175</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDeux">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>225</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePhone">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>75</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="leID">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>25</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblState1">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>75</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblState2">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>175</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblState3">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>225</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>275</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>275</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销改动</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>125</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>125</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblState4">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>125</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbOrders">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>25</y>
     <width>450</width>
     <height>600</height>
    </rect>
   </property>
   <property name="title">
    <string>查看历史订单</string>
   </property>
   <widget class="QTableView" name="tbOrders">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>150</y>
      <width>400</width>
      <height>425</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblOrders">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>请选择订单号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbOrders">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbComments">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>400</y>
     <width>450</width>
     <height>225</height>
    </rect>
   </property>
   <property name="title">
    <string>查看历史评论</string>
   </property>
   <widget class="QTableView" name="tbComments">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>30</y>
      <width>400</width>
      <height>175</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lePhone</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ProfileView</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePassword</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ProfileView</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDeux</sender>
   <signal>textEdited(QString)</signal>
   <receiver>ProfileView</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>ProfileView</receiver>
   <slot>OnSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUndo</sender>
   <signal>clicked()</signal>
   <receiver>ProfileView</receiver>
   <slot>OnUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOrders</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ProfileView</receiver>
   <slot>BindOrder(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnDelta()</slot>
  <slot>OnSave()</slot>
  <slot>OnUndo()</slot>
  <slot>BindOrder(QString)</slot>
 </slots>
</ui>
